src/dr/app/beauti/options/PartitionTreeModel.java

```diff
@@ -155,7 +192,11 @@ public List<Parameter> selectParameters(List<Parameter> parameters) {
 
         // Don't add these to the parameter list (as they don't appear in the table), but call
         // get parameter so their id prefix can be set.
+        
         getParameter("tree");
+        if(isUsingEmpiricalTrees()){
```

Comment:
This is changing the logic for isUsingEmpiricalTrees == true, is this correct?

Reply:
we can remove this. I didn't think empirical trees had these parameters since the only operator was a tree swap operator. But maybe there are other uses for these parameters. Happy to remove this if my reasoning is off!